.github/workflows/scheduled.yml

```
@@ -124,10 +124,10 @@ jobs:
           brew install curl
           echo "HOMEBREW_FORCE_BREWED_CURL=1" >>"${GITHUB_ENV}"
 
-      - name: Install and use Homebrew svn if needed
+      - name: Install Homebrew subversion if needed for downloading sources
         run: |
-          if brew ruby -e 'exit 1 if Formula[ARGV.first].deps.none? { |d| d.name == "subversion" && d.implicit? }' "$FORMULA"; then
-            brew install svn
+          if brew deps --include-implicit -1 $FORMULA | grep -q subversion; then
```

```suggestion
          if brew deps --include-implicit --direct "$FORMULA" | grep -Fqx subversion; then
```

I suggest that we:

- Quote the `$FORMULA` variable, or actionlint will likely complain.
- Use the long option `--direct` so that it's hopefully more easy to understand than the shorthand `-1`.
- Use the `-F` (use literal strings) and `-x` (force pattern to match only whole lines) flags of `grep` too.